server/index.js

```diff
@@ -100,9 +100,10 @@ export async function createServer(
       });
       res.status(200).send(data);
     } catch (e) {
-      res
-        .status(500)
-        .send({ message: 'There was an error fulfilling the order' });
+      console.log(e.message);
```

Comment:
Was getting a 500 that I couldn't track down - turns out i had my inventory configured wrong, so now im sending this error message to the front end. Although the format isnt very nice because these 422s get wrapped up in a string array.